.helm/ecamp3/templates/frontend_configmap.yaml

```diff
@@ -18,4 +18,5 @@ data:
       DEPLOYMENT_TIME: '{{ .Values.deploymentTime }}',
       VERSION: '{{ .Values.deployedVersion }}',
       VERSION_LINK_TEMPLATE: '{{ .Values.versionLinkTemplate }}',
+      RECAPTCHA_SITE_KEY: '{{ .Values.recaptcha.siteKey }}',
```

Comment:
Das Feature Branch Deployment failt noch, wenn in den values kein `recaptcha` drin ist. Also entweder in values.yml noch leer ergänzen, oder hier ein `if` einfügen wie es beim Sentry DSN schon gemacht ist. die ReCaptcha-Anbindung ist ja genau wie Sentry optional.